webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
```

Comment:
- [ ] . Ensure tha Webpack devserver is setup and running with no errors. I cannot run this on my local machine. Kindly take a minute and go through the webpack exercise you have already completed if you need a refresher
- [ ] Kindly ensure that there is a html file configured in the Webpack config settings as a template using the *HTML Webpack Plugin*.
- [ ] Also ensure that CSS must be loaded by *Webpack Style/CSS Loader*.